src/foam/nanos/dig/SUGAR.js

```diff
@@ -208,7 +208,7 @@ foam.CLASS({
       // TODO: appears not to work if named 'url', find out why.
       name: 'sugarURL',
       label: 'URL',
-      hidden: false,
+      hidden: true,
```

Comment:
Why do we hide this?

Reply:
Since we adopt the post method for all SUGAR service. The URL will be not useful. Instead, the bottom will send the request.

Reply:
looks like you are using the URL below to perform the POST, no?